src/main/java/blusunrize/immersiveengineering/common/blocks/metal/MetalPressTileEntity.java

```diff
@@ -164,7 +166,9 @@ public void onEntityCollision(World world, Entity entity)
 			if(recipe==null)
 				return;
 			ItemStack displayStack = recipe.getDisplayStack(stack);
-			float transformationPoint = 56.25f/120f;
+//			float processMaxTicks = recipe.getTotalProcessTime();
+//			float transformationPoint = (MetalPressRenderer.getPressTime(processMaxTicks)+MetalPressRenderer.getPressTime(processMaxTicks))/processMaxTicks;
+			float transformationPoint = 0.5f;
```

Comment:
Why's there commented out code here?

Reply:
I left it there in case there's a reason to put the transformationPoint to the moment where the press piston hits the conveyor
I put in 0.5f because it required no calculation and calls and still works for all recipes

I can remove/change it if you want